pkg/omf/functions/cli/omf.cli.describe.fish

```diff
@@ -1,4 +1,12 @@
-function omf.cli.describe -a name
-  omf.packages.describe $name
-  return 0
+function omf.cli.describe -a package -d 'Show information about a package'
+  if set -l props (omf.index.stat $package description repository maintainer)
+    echo "Package: $package"
+    echo "Description: $props[1]"
+    echo "Repository: $props[2]"
+    echo "Maintainer: $props[3]"
```

Comment:
Don't worry, the `omf.index.stat` command will return a blank line for property names not found. This works when any field is missing and just shows an empty string as the value.